### ISSUE TITLE
Clarification on Project Scope and Vision

### ISSUE THREAD

**Issue by thompsondt** [CurtisGreen/opensidewalkmap#11]
Hi @CurtisGreen,

I hope you're doing well. I'm interested in contributing new features to OpenSidewalkMap and would like to understand the project's scope and long-term vision better. So far, I've contributed some documentation updates and opened a PR to address viewing sidewalks marked as features of the road.

I'm considering the following improvements:

- UX refinements for desktop and mobile (updating the layout to support additional details and user activities)
- Features for sidewalk data collection (uploads data for OSM)
- Features for statistical breakdown of Active Transportation Infrastructure assets for compliance and inventory
- Deployment as a progressive web app or app store app for mobile users

I may only have time to design these features for now but would like to eventually deliver a working version of them. If these features don't align with OpenSidewalkMap's vision, I might consider developing them as a standalone project.

Could you please clarify if these align with your vision and scope for OpenSidewalkMap?

Thanks for your time and for maintaining this project. I look forward to your response.

Best,
David

**Comment by CurtisGreen** [comment]
> UX refinements for desktop and mobile (updating the layout to support additional details and user activities)

I'd need more specifics but probably fine

> Features for sidewalk data collection (uploads data for OSM)

I think as long as it's focused to sidewalks this could be good

> Features for statistical breakdown of Active Transportation Infrastructure assets for compliance and inventory

Things like length of sidewalk within the area would be doable with the current data, open parking map does that for parking area and I think it's cool. I just hadn't implemented it yet

> Deployment as a progressive web app or app store app for mobile users

I generally prefer making the website as responsive as possible for mobile but I understand some people enjoy apps. If it doesn't drastically change/add code then I think this would be fine, otherwise maybe should be separate and can just be updated whenever the site updates

TLDR I want it to be very focused on pedestrian infrastructure particularly sidewalks. Anything related to that would be great and anything outside of that I would need to consider

**Comment by thompsondt** [comment]
UX refinements:
- I'm producing some wireframes that illuminate at least part of that.

Data Collection:
- imagine https://github.com/OPENER-next/OpenStop, but data collection scope includes curbs, sidewalks, crossings
- this information is expected to support prioritization of accessibility improvements and alignment of key active transportation routes in my region

Statistics:
- data collection progress, ex:
  - Overpass detects 741 tagged and _potential_ curbs in the area (ex: https://overpass-turbo.eu/s/1O5l )
  - 110 curbs have been tagged, users can help us document the other 631 with rich data
- breakdown of curbs and compliance -- information useful for planners, advocates, and decision makers
- breakdown of sidewalks, crossings, and their compliance

Deployment:
- We might get pretty far with a PWA and responsive design before the need to ship through the app store

I can ping you with some more complete details. I'd like to get your feedback and ideas before doing any higher-fidelity prototyping. There is a bit of a backlog (story map) forming. I can share that with you as well.

**Comment by CurtisGreen** [comment]
I'd be okay with it as long as the features are different enough from OpenStop or any other known application.

Currently I'd say the main features of open sidewalk map are:
- It's simple
- It's up to date by dynamically grabbing the latest data
